netbout-web/src/main/java/com/netbout/rest/bout/TkAttach.java

```diff
@@ -133,7 +122,27 @@ public Response act(final Request req) throws IOException {
             throw new RsFailure(ex);
         }
         FileUtils.forceDelete(temp);
-        throw new RsForward(new RsFlash(msg.toString()));
+        bout.messages().post(msg.toString());
+        return new RsForward(new RsFlash(msg.toString()));
```

Comment:
@bdragan not sure about this change since it changes the previous behavior. can't we keep the throw?